UpdateLeadStatus.cls

```diff
@@ -0,0 +1,13 @@
+/*Find all the Lead records having Status as Won & update the stage with New value*/
+
+public class UpdateLeadStatus{
+	public static void setStatus(){
+		String newStatus = 'Closed - Converted';
+		List<Lead> leadList = new List<Lead>();
+		List<Lead> updateLead = [SELECT Id, Name,Status FROM Lead where Status='Open - Not Contacted'];
+		for (Lead leadList1: updateLead){
+        	leadList1.Status=newStatus;
+        	update LeadList1;

```

Comment:
Why `update` statement is inside for loop? 

- [ ] Remove update statement form for loop